### PR TITLE
lbheartbeat and version endpoints for all services

### PR DESCRIPTION
**Pull request by matt-boris**

### CODE REVIEW

File: libraries/api/src/index.js [hunk 1]
```diff
@@ -32,6 +36,53 @@ const ping = {
...
+  title: 'Taskcluster Version',
+  category: 'Taskcluster Version',
+  description: [
+    'Respond with the JSON version object.',
+    'https://github.com/mozilla-services/Dockerflow/blob/main/docs/version_object.md',
+  ].join('\n'),
+  handler: function(_req, res) {
+    try {
+      const taskclusterVersionFile = path.resolve(REPO_ROOT, 'version.json');
+      const taskclusterVersion = fs.readFileSync(taskclusterVersionFile).toString().trim();
```
Comment (lotas) [inline libraries/api/src/index.js:78]: this should be done outside of the handler. `version.json` will not be changed during whole deployment, so there is no need to do a blocking read inside the handler, and pause event loop for other requests.

Comment (lotas) [reply libraries/api/src/index.js:78]: or at least, if this is dynamic (which is probably not) we can do an async call here to avoid blocking event loop

Comment (matt-boris) [reply libraries/api/src/index.js:78]: Good call, thanks @lotas

**Review by lotas — APPROVED** [review]

**Review by lotas — APPROVED** [review]

**Comment by lotas** [comment]
tested on dev cluster one of the services, it returns `__version__` and `__lbheartbeat__`

**Pull request merged**